libraries/core_libs/network/src/tarcap/packets_handlers/common/ext_votes_packet_handler.cpp

```diff
@@ -34,13 +36,17 @@ void ExtVotesPacketHandler::sendPbftVote(dev::p2p::NodeID const &peer_id, std::s
 }
 
 void ExtVotesPacketHandler::sendPbftNextVotes(dev::p2p::NodeID const &peer_id,
-                                              std::vector<std::shared_ptr<Vote>> const &send_next_votes_bundle) {
+                                              std::vector<std::shared_ptr<Vote>> const &send_next_votes_bundle,
+                                              uint32_t index) {
   if (send_next_votes_bundle.empty()) {
     return;
   }
 
-  dev::RLPStream s(send_next_votes_bundle.size());
-  for (auto const &next_vote : send_next_votes_bundle) {
+  uint32_t vote_count_to_send = std::min((size_t)kMaxVotesInPacket, send_next_votes_bundle.size() - index);
```

Comment:
(size_t) is old style casting, static_cast<> is preffered

Reply:
Fixed